pkg/s3wrapper/filesystem_test.go

```diff
@@ -157,16 +157,47 @@ var _ = Describe("s3filesystem", func() {
 	})
 	Context("upload boot files", func() {
 		It("all exist", func() {
+			err := os.MkdirAll(filepath.Join(baseDir, "files/images/pxeboot"), 0755)
+			Expect(err).ToNot(HaveOccurred())
+			err = ioutil.WriteFile(filepath.Join(baseDir, "files/images/pxeboot/rootfs.img"), []byte("this is rootfs"), 0600)
+			Expect(err).ToNot(HaveOccurred())
+			err = ioutil.WriteFile(filepath.Join(baseDir, "files/images/pxeboot/initrd.img"), []byte("this is initrd"), 0600)
+			Expect(err).ToNot(HaveOccurred())
+			err = ioutil.WriteFile(filepath.Join(baseDir, "files/images/pxeboot/vmlinuz"), []byte("this is vmlinuz"), 0600)
+			Expect(err).ToNot(HaveOccurred())
+			err = os.MkdirAll(filepath.Join(baseDir, "files/EFI/redhat"), 0755)
+			Expect(err).ToNot(HaveOccurred())
+			err = ioutil.WriteFile(filepath.Join(baseDir, "files/EFI/redhat/grub.cfg"), []byte(" linux /images/pxeboot/vmlinuz"), 0600)
+			Expect(err).ToNot(HaveOccurred())
+			err = os.MkdirAll(filepath.Join(baseDir, "files/isolinux"), 0755)
+			Expect(err).ToNot(HaveOccurred())
+			err = ioutil.WriteFile(filepath.Join(baseDir, "files/isolinux/isolinux.cfg"), []byte(" append initrd=/images/pxeboot/initrd.img"), 0600)
+			Expect(err).ToNot(HaveOccurred())
+			mockVersions.EXPECT().GetRHCOSVersion(defaultTestOpenShiftVersion).Return(defaultTestRhcosVersion, nil).Times(1)
+			srcObject, err := client.GetBaseIsoObject(defaultTestOpenShiftVersion)
+			Expect(err).ToNot(HaveOccurred())
+			err = ioutil.WriteFile(filepath.Join(baseDir, "files/images/assisted_installer_custom.img"), make([]byte, isoeditor.RamDiskPaddingLength), 0600)
+			Expect(err).ToNot(HaveOccurred())
+			err = ioutil.WriteFile(filepath.Join(baseDir, "files/images/ignition.img"), make([]byte, isoeditor.IgnitionPaddingLength), 0600)
+			Expect(err).ToNot(HaveOccurred())
+			isoPath := filepath.Join(baseDir, srcObject)
+			cmd := exec.Command("genisoimage", "-rational-rock", "-J", "-joliet-long", "-V", "volumeID", "-o", isoPath, filepath.Join(baseDir, "files"))
+			err = cmd.Run()
+			Expect(err).ToNot(HaveOccurred())
+			err = os.RemoveAll(filepath.Join(baseDir, "files"))
```

Comment:
nit: Should this be in an  `AfterEach` block?

If the test fails, this has the potential to leave files behind. Space is cheap, but not unlimited :stuck_out_tongue: 


Actually, unless I'm missing something, it looks like `baseDir` is not being cleaned after the test suite runs:

https://github.com/openshift/assisted-service/blob/d8ce8ec53d9641be697f25b2f218f39b59c460d7/pkg/s3wrapper/filesystem_test.go#L37

We may want to add an `AfterSuite` blockthat removes it:

https://github.com/openshift/assisted-service/blob/d465cf39a1efc293dc45a28468f9b9b6e23855fb/internal/isoeditor/rhcos_test.go#L51-L58

Reply:
Left what exists alone and added a clean of the entire `baseDir` to the `AfterEach()`.